Patterns/Chinese/Chinese-DateTime.yaml

```diff
@@ -108,10 +108,10 @@ RelativeMonthRegex: !nestedRegex
   def: (?<relmonth>({DatePeriodThisRegex}|{DatePeriodLastRegex}|{DatePeriodNextRegex})\s*月)
   references: [DatePeriodThisRegex, DatePeriodLastRegex, DatePeriodNextRegex]
 DatePeriodYearRegex: !nestedRegex
-  def: (({YearNumRegex})(\s*年)?|({YearRegex})\s*年)(?=[\u4E00-\u9FFF]|\s|$|\W)
+  def: ((({YearNumRegex})(\s*年)?|({YearRegex})\s*年))
```

Comment:
Remove duplicated parentheses

Reply:
It is necessary otherwise the Python won't work.

Reply:
Sorry for this.